backend/vnc.c

```diff
@@ -0,0 +1,246 @@
+/*
+Twin - A Tiny Window System
+Copyright (c) 2024 National Cheng Kung University, Taiwan
+All rights reserved.
+*/
+#include <aml.h>
+#include <assert.h>
+#include <neatvnc.h>
+#include <pixman.h>
+#include <stdlib.h>
+#include <string.h>
+#include <twin.h>
+#include "twin_backend.h"
+#include "twin_private.h"
+
+#define SCREEN(x) ((twin_context_t *) x)->screen
+#define PRIV(x) ((twin_vnc_t *) ((twin_context_t *) x)->priv)
+#define VNC_HOST "127.0.0.1"
```

Comment:
Consider allowing the user to specify the IP address and port via environment variables.
Check: https://github.com/sysprog21/mado/blob/main/backend/fbdev.c#L189